alwayson/app.py

```diff
@@ -10,7 +10,7 @@
 
 # Setup AlwaysON System Routes
 # The sys_router for is for Always ON System Routes
-sys_router = APIRouter(prefix='/v1')
+sys_router = APIRouter(prefix="/v1")
```

Comment:
oooh I think we're going to have a philosophical difference here. I like the single quotes until I need double quotes.
(i.e. `"This isn't doable with single quotes"`

Reply:
That being said I'm fine using Black but I'm surprised it didn't catch more instances where I was using single-quotes.

Reply:
Nice catch, I'll see if there's something I missed

Reply:
I just did a search and I'm only seeing double quotes in the `.py` files.

Reply:
Yea all looks good. Must have been a glitch as to why I was only seeing this change 🤷